src/Components/Header/Header.js

```diff
@@ -79,10 +79,19 @@ export default function Header() {
                   pathProps={logoutIconProps}
                 />
               </button>
-              <div className="flex items-center cursor-pointer">
-                <Link to={`/p/karl`}>
-                  <Image {...profileImageProps}></Image>
-                </Link>
+              <div className="flex flex-col items-center cursor-pointer">
+                <img
+                  src="/images/avatars/karl.jpg"
+                  alt="Profile"
+                  className="rounded-full h-8 w-8 flex"
+                  onClick={handleImageClick}
+                />
+                {isDropdownVisible && (
+                  <div className="dropdown-content absolute mt-6 flex flex-col">
+                    <Link to={`/profile`}>Profile</Link>
+                    <Link to={`/settings`}>Settings</Link>
```

Comment:
1. Use an array to return one `<Link/>`
2. They should be a button, look how they look when you display them
![image](https://github.com/waleedtarbosh/Instagram-Clone/assets/81507185/f294c08b-1d62-42da-afaf-c872dbc258d3)
3. Why are you not using routes.js?